cg-charter.html

```diff
@@ -224,6 +226,15 @@ <h3 id="test-suites">
       "http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">
       W3C Software and Document License</a>.
     </p>
+    <p>
+      The Community Group will develop a continuous integration mechanism to validate the
+      grammar of WebGPU Shading Language against syntactic ambiguities and potential
+      mechanical issues with primary reliance on an industry-adopted tool that can report
```

Comment:
I'm not sure how to read "with primary reliance on an industry-adopted tool". That seems to leave the door open to a mechanism based on a closed-source and commercial tool, provided it is widely adopted by the industry. Is that intended? Or is the goal rather to build on top of an open-source and free-to-use tool so that the community can leverage this work more easily?

Reply:
@tidoust Exactly, I am sorry for wording there. Would explicitly mentioning open-source and such aim help?

Reply:
@tidoust I've tried to rephrase a bit